Lib/test/test_codecs.py

```diff
@@ -3440,5 +3440,22 @@ def search_function(encoding):
         self.assertEqual(NOT_FOUND, codecs.lookup('a\xe9\u20ac-8'))
 
 
+class EncodingNormalizationTest(unittest.TestCase):
+
+    def test_bpo39337(self):
+        """
+        bpo-39337: similar to _Py_normalize_encoding(),
+        encodings.normalize_encoding() should ignore non-ASCII letters.
+        """
+        import encodings
```

Comment:
it's already imported at the top-level.

Reply:
Oh, thank you. Copy from test_source_encodings.py directly~